Configurateur/front/lib/components/tickets_page.dart

```diff
@@ -0,0 +1,929 @@
+                                                            lastTicket[
+                                                                "createdAt"],
+                                                          )}",
+                                                          style: TextStyle(
+                                                            color:
+                                                                getCurrentTheme()
+                                                                    .primaryColor,
+                                                          ),
+                                                        ),
+                                                      ],
+                                                    ),
+                                                  ),
+                                                ),
+                                                isAdmin && notAssigned(tickets)
+                                                    ? Align(
+                                                        alignment: Alignment
+                                                            .centerRight,
+                                                        child: GestureDetector(
+                                                          onTap: () async {
+                                                            await assignTicket(
+                                                                tickets:
+                                                                    tickets);
+                                                          },
+                                                          child: const Text(
+                                                            "S'assigner",
+                                                            style: TextStyle(
+                                                              color:
+                                                                  Colors.blue,
+                                                            ),
+                                                          ),
+                                                        ),
+                                                      )
+                                                    : Container()
+                                              ],
+                                            ),
+                                          ),
+                                        );
+                                      },
+                                    )
+                            ],
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  const Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 32),
+                    child: SizedBox(
+                      height: 512,
+                      width: 1,
```

Comment:
responsive

Reply:
not handled

Reply:
width cannot be handled because we want a FIX thickness of the vertical divider

Reply:
height handled